main.js

```diff
@@ -0,0 +1,37 @@
+const tweeter = Tweeter()
+const renderer = Renderer()
+
+renderer.renderPosts(tweeter.getPosts())
+
+
+const twit = $('#post')
+const inputPost = $('input')
+
+const post = function () {
+    if (inputPost.val() != "") {
```

Comment:
make sure you always use `===` and `!==`

this is because you will run into problems in more complicated code. Look at the link below to see the difference :)
https://www.w3schools.com/js/js_comparisons.asp